### DISCUSSION TITLE
K9 mail wrong display

### DISCUSSION THREAD

**Discussion by Spyros2455** [community/community#112500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Good evening dear all
emails from Greek telephone provider display with unknown characters.
Please see attachements S_1 =Wrong, S_2 Correct.
Best regards
Spyros
![S_1](https://github.com/community/community/assets/58724816/ead84054-221d-46f6-b872-cd62ea55a67c)
![S_2](https://github.com/community/community/assets/58724816/59a3208f-b062-4d26-ad9d-11f2a02f7dd5)

**Comment by ettaboyle** [comment]
Hi @Spyros2455,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
